metabase/metabase
# Add extended color pickers

Epic https://github.com/metabase/metabase/issues/22816

Adds new `ColorPicker` and `ColorSelector`. To be used to pick a color from the new color palette (in a follow-up PRs).

How to test:
- Open storybook `yarn storybook`
- Make sure it's possible to select a color with both `ColorPicker` and `ColorSelector`.

Please note that fonts are broken in storybook atm.

<img width="817" alt="Screenshot 2022-05-23 at 22 09 44" src="https://user-images.githubusercontent.com/8542534/169889678-4628f09b-c52f-4688-8632-27df23bd3857.png">
<img width="871" alt="Screenshot 2022-05-23 at 22 09 58" src="https://user-images.githubusercontent.com/8542534/169889687-0a43b948-169c-474f-b24e-ee2bcd1479ed.png">
